front/src/config/i18n/fr.json

```diff
@@ -791,6 +791,18 @@
         "magnet": "Capteur magnétique"
       }
     },
+    "oauth2": {
+      "buttonConnect": "Connecter maintenant",
+      "apiKeyLabel": "Saisissez ensuite votre clé client_id et secret_id ici:",
+      "errorEmptyConfig": "Client Id et Secret Id ne doivent pas être vide !",
+      "errorServiceId": "Service Id de l'intégration courante n'a pas été trouvé !",
```

Comment:
Same as in english, not sure this case exists.

Reply:
this case exist only if developer who use this part does a mistake